gunagala/imager.py

```diff
@@ -1561,11 +1578,11 @@ def make_noiseless_image(self,
         """
         electrons = np.zeros((self.wcs._naxis2,
                               self.wcs._naxis1)) * u.electron / (u.second * u.pixel)
+        self.set_WCS_centre(centre)
 
         # Calculate observed sky background
         sky_rate = self.sky_rate[filter_name]
         if hasattr(self.sky, 'relative_brightness'):
-            pixel_coords = self.get_pixel_coords(centre)
```

Comment:
This line needs to be reinstated now, but without the `centre` argument as the `set_WCS_centre` call is taking care of that now.